src/components/LinkEditor.tsx

```diff
@@ -35,14 +36,18 @@ type Props = {
 
 class LinkEditor extends React.Component<Props> {
   discardInputValue = false;
-  initialValue: string = this.props.href;
+  initialValue = this.href;
 
   state = {
     selectedIndex: -1,
-    value: this.props.href,
+    value: this.href,
     results: [],
   };
 
+  get href(): string {
+    return this.props.mark ? this.props.mark.attrs.href : null;
+  }
```

Comment:
@tommoor lmk what you think of this pattern. Consolidating the null check into a single getter

Reply:
Works for me 👍  (Although your return type should be `?string` here)

Reply:
changed to empty string to make it play nicer with inputs